src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/services/NodeService.kt

```diff
@@ -223,4 +118,135 @@ class NodeService(
       is TransactionWithClosureError -> getPaymentTypeCode(tx.transactionAtPreviousState)
       else -> TIPO_VERSAMENTO_CP
     }
+
+  private fun buildClosePaymentForCancellationRequest(
+    transactionWithCancellation: BaseTransactionWithCancellationRequested,
+    transactionOutcome: ClosePaymentRequestV2Dto.OutcomeEnum,
```

Comment:
This must be always equal to `ClosePaymentRequestV2Dto.OutcomeEnum.KO`, so either:

* check this invariant inside this method
* remove this parameter and replace it with the constant

Reply:
Fixed with 👉🏻 3740909e21260868428e86bf4c0fcae1dd87fb71